plugins/filters/bytedelta/bytedelta.h

```diff
@@ -8,7 +8,9 @@
   See LICENSE.txt for details about copyright and rights to use.
 **********************************************************************/
 
-#pragma once
+#ifndef BLOSC_PLUGINS_FILTERS_BYTEDELTA_BYTEDELTA_H
+#define BLOSC_PLUGINS_FILTERS_BYTEDELTA_BYTEDELTA_H
```

Comment:
Duplicated BYTEDELTA substring?

Reply:
Duplicate as the path itself:
`plugins/filters/`**`bytedelta/bytedelta.h`**

Reply:
I see the pattern now.  It is ok with me.